core/controllers/learner_playlist.py

```diff
@@ -27,9 +27,33 @@
 class LearnerPlaylistHandler(base.BaseHandler):
     """Handles operations related to the learner playlist."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'activity_type': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
```

Comment:
Limit this only to allowed activity types.

Reply:
Done